lib/gs-app.c

```diff
@@ -4032,6 +4041,14 @@ gs_app_class_init (GsAppClass *klass)
 				     G_PARAM_READWRITE | G_PARAM_CONSTRUCT);
 	g_object_class_install_property (object_class, PROP_QUIRK, pspec);
 
+	/**
+	 * GsApp:key-colors:
+	 */
+	pspec = g_param_spec_string ("key-colors", NULL, NULL,
```

Comment:
I think you want this to be a `g_param_spec_pointer`? Otherwise how can the calls to `g_value_get_pointer()` and `g_value_set_pointer()` above work at all?

Reply:
Wow. Sorry about that. I am surprised it all apparently worked...